awx/ui_next/src/screens/Team/TeamAccess/TeamAccessListItem.jsx

```diff
@@ -40,9 +40,6 @@ function TeamAccessListItem({ role, i18n, detailUrl, onSelect }) {
                     label={i18n._(t`Role`)}
                     value={
                       <Chip
-                        isReadOnly={
```

Comment:
Was this removed intentionally?  We probably need to keep it so that only those that have permission to remove the item are given the button to do so.

Reply:
it looks like the chips are failing tests too.  i'll take a look at those in just a second

Reply:
i had to add these back in.  their code mods deleted them erroneously